go.mod

```diff
@@ -3,20 +3,23 @@ module github.com/grpc-ecosystem/protoc-gen-grpc-gateway-ts
 go 1.14
 
 require (
-	github.com/Masterminds/goutils v1.1.0 // indirect
+	github.com/Masterminds/goutils v1.1.1 // indirect
 	github.com/Masterminds/semver v1.5.0 // indirect
 	github.com/Masterminds/sprig v2.22.0+incompatible
-	github.com/golang/protobuf v1.4.3
-	github.com/grpc-ecosystem/grpc-gateway v1.15.2
+	github.com/golang/protobuf v1.5.2
+	github.com/google/uuid v1.2.0 // indirect
+	github.com/grpc-ecosystem/grpc-gateway/v2 v2.3.0
```

Comment:
Although it's good to know this generator is working with v2 version of `grpc-gateway`, could we please don't make a sudden major version change like this? For reasons we are still using `grpc-gateway` v1, and it will be for a while. 

The support for different `grpc-gateway` versions in test is something needs to think about. And should be left out for this PR.

Reply:
I understand. My apologies, I will revert these changes 🙇🏼